### DISCUSSION TITLE
Coldtype on EYESY

### DISCUSSION THREAD

**Discussion by bitzky** [coldtype/coldtype#81] [Ideas]
Did someone install COLDTYPE on @critterandguitari EYESY? is it possible? This would be a perfect match!

**Comment by stenson** [comment]
Interesting! Could definitely be possible, though from my experience running Coldtype on lower-powered processors (like a Chromebook), it's pretty difficult to get real-time happening there with anything more than a couple letters, since Coldtype is more optimized for precision (ability to manipulate glyph shapes) than for speed, which would probably be necessary for a good experience on EYESY
